starter-code/clue.js

```diff
@@ -1,104 +1,147 @@
+  color: "red",
+  description: "She is an A-list movie star with a dark past",
   age: 31,
-  image: https://metrouk2.files.wordpress.com/2016/07/miss-scarlett.jpg,
-  occupation: Actor
-
- Mrs Peacock
-  first name: Eleanor
-  last name: Peacock
-  color: blue
-  description: She is from a wealthy family and uses her status and money to earn popularity
-  age: 36
-  image: https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg
-  occupation: Socialité
-
- Mr Mustard
-  first name: Jack
-  last name: Mustard
-  color: yellow
-  description: He is a former football player who tries to get by on his former glory
-  age: 62
-  image: https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg
-  occupation: Retired Football player
-
----- Weapons ----
-
-Rope - Weight: 10
-Knife - Weight: 8
-Candlestick - Weight: 2
-Dumbbell - Weight: 30
-Poison - Weight: 2
-Axe - Weight: 15
-Bat - Weight: 13
-Trophy - Weight: 25
-Pistol - Weight: 20
-
-
----- Rooms ----
-Dinning Room
-Conservatory
-Kitchen
-Study
-Library
-Billiard Room
-Lounge
-Ballroom
-Hall
-Spa
-Living Room
-Observatory
-Theater
-Guest House
-Patio
-
-*/
+  image: "https://metrouk2.files.wordpress.com/2016/07/miss-scarlett.jpg",
+  occupation: "Actor",
+},
+{
+  firstName: "Eleanor",
+  lastName: "Peacock",
+  color: "blue",
+  description: "She is from a wealthy family and uses her status and money to earn popularity",
+  age: 36,
+  image: "https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg",
+  occupation: "Socialité",
+},
+ {
+  firstName: "Jack",
+  lastName: "Mustard",
+  color: "yellow",
+  description: "He is a former football player who tries to get by on his former glory",
+  age: 62,
+  image: "https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg",
+  occupation: "Retired Football player",
+},
+];
+
+
+//Weapons ----
+
+var weapons = [
+    {
+  name: "rope",
+  weight: 10,
+},
+{
+  name: "knife",
+  weight: 8,
+},
+{
+  name: "candlestick",
+  weight: 2,
+},
+{
+  name: "dumbbell",
+  weight: 30,
+},
+{
+  name: "poison",
+  weight: 2,
+},
+{
+  name: "axe",
+  weight: 15,
+},
+{
+  name: "bat",
+  weight: 13,
+},
+{
+  name: "trophy",
+  weight: 25,
+},
+{
+  name: "pistol",
+  weight: 20,
+},
+    ];
+
+//Rooms ----
+
+var rooms = [
+{ name: "Dinning Room"},
+{ name: "Conservatory"},
+{ name: "Kitchen"},
+{ name: "Study"},
+{ name: "Library"},
+{ name: "Billiard Room"},
+{ name: "Lounge"},
+{ name: "Ballroom"},
+{ name: "Hall"},
+{ name: "Spa"},
+{ name: "Living Room"},
+{ name: "Observatory"},
+{ name: "Theater"},
+{ name: "Guest House"},
+{ name: "Patio"},
+];
 
 function random_selector(array) {
-
+  var index = Math.floor(Math.random()*array.length);
+  return array[index];
 }
+//console.log(random_selector(characters));
 
-function pick_mistery() {
 
+function pick_mistery() {
+var mistery = {};
+mistery.killer = random_selector(characters);
+mistery.room = random_selector(rooms);
+mistery.weapon = random_selector(weapons);
+return mistery;
```

Comment:
You can also do something like this if you want:

```
mystery: {
  killer: random_selector(characters),
  location: random_selector(rooms),
  weapon: random_selector(weapons)
}
```

By the way, I liked that you named the property `killer`instead of `character`. It has much more meaning.